cli/targets/static.js

```diff
@@ -354,8 +354,48 @@ function toJsType(field) {
     return type;
 }
 
+function syntaxForType(type) {
+
+    var syntax = null;
+    var namespace = type;
+
+    while (syntax === null && namespace !== null) {
+        if (namespace.options != null && "syntax" in namespace.options)
```

Comment:
For readability, please wrap if/else statements in curly brackets

Reply:
Done, changes pushed